modules/phase_field/doc/content/source/ics/CoupledValueFunctionIC.md

```diff
@@ -0,0 +1,30 @@
+# CoupledValueFunctionIC
+
+!syntax description /ICs/CoupledValueFunctionIC
+
+CoupledValueFunctionIC initializes the value of a variable with the value of a
+coupled  [MOOSE Function](functions.md) which is evaluated with a set of up to
+four coupled  variable values `v` as its input parameters. The coupled variable
+values are sunstituted for the  `x`,`y`,`y`, and `t` function variables in that
```

Comment:
```suggestion
values are sunstituted for the  `x`,`y`,`z`, and `t` function variables in that
```

Reply:
Lol did you even look at my review?

Reply:
I think so?
I didnt report the typo you reported